widgetssdk/src/main/java/com/glia/widgets/chat/ChatManager.kt

```diff
@@ -140,12 +145,19 @@ internal class ChatManager(
 
     @VisibleForTesting
     fun checkUnsentMessages(state: State) {
-        val unsent = state.unsentItems.firstOrNull { it.error == null } ?: return
-        sendUnsentMessagesUseCase(unsent, {
-            onChatAction(Action.MessageSent(it))
+        val payload = state.preEngagementChatItemIds.takeIf { it.isNotEmpty() }
+            ?.firstOrNull()
+            ?.let { state.messagePreviews[it] } ?: return
+
+        sendMessage(payload)
+    }
+
+    private fun sendMessage(payload: SendMessagePayload) {
+        println("*********************** -> ${payload.content}")
```

Comment:
@DavDo Don't forget to remove this line. 😉

Reply:
thanks